Open-Science-Workshop-2022/tutorials/act-basics.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>I think having a header for Data Cleaning, First Look, QC, and final viz would be helpful!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ARM-Development/ARM-Notebooks/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>